src/orquestra/sdk/_ray/_ray_logs.py

```diff
@@ -64,7 +150,15 @@ def _refine_log_line(self, line):
         elif (
             self.workflow_or_task_run_id is None
         ) or self.workflow_or_task_run_id in line:
-            return line
+            if line.startswith("{"):
+                log = self._load_structured_log_record(line)
+            else:
+                log = self._load_unstructured_log_record(line)
+
+            if log is None:
+                return
+
+            return LINE_FORMAT.format(log=log).replace("[]", "").strip()
```

Comment:
**question**: `.format()` followed by a `.replace("[]", "")`. Seems redundant. Can the format string be changed so get the right output in one step?